### ISSUE TITLE
List: screen reader doesn't announce selected element

### ISSUE THREAD

**Issue by wkeese** [ibm-js/delite#123]
Try list_singleSelection_markAfter.html using JAWS 14.    The screen reader never announces which item(s) are selected.  Thus, blind users have no way of knowing which item(s) are selected.

Contrast with the HTML below, where the selected item is announced when the list is focused:

``` xml
<div role="listbox" tabindex="0">
    <div role="option" tabindex="0">goat</div>
    <div role="option" aria-selected="true" tabindex="0">frog</div>
    <div role="option" tabindex="0">horse</div>
</div>
```

I don't know how to get JAWS to announce which item(s) are selected without focusing the List itself, but the current code is not usable (if you don't have vision).
